app/logic/Mail/ActiveSync/ActiveSyncAccount.ts

```diff
@@ -117,6 +117,20 @@ export class ActiveSyncAccount extends MailAccount {
     if (email.bcc.hasItems) {
       throw new NotSupported("ActiveSync does not support BCC");
     }
+    assert(email.folder?.id, "Need folder to save the sent email in");
+    let defaultSentFolder = this.getSpecialFolder(SpecialFolder.Sent) as ActiveSyncFolder;
```

Comment:
Note that the user can change the Sent folder in settings. In this case, the Sent folder flag does not match the server's idea of what the Sent folder is.

Reply:
Well, this very PR would allow us to remove that limitation.

No assumptions, remember?

Reply:
TODO: When listing folders, remember (in a variable) the sent folder that the server told us, and use that sent folder here. That allows the user to set a custom sent folder.